cv.tex

```diff
@@ -44,92 +50,85 @@
 % Education
 %------------------------------------------------
 
-\section{Education} 
-
-\subsection{Cornell University}
-
-\descript{MEng in Computer Science}
-\location{Expected Dec 2014 | Ithaca, NY \\ Cum. GPA: N/A}
+\section{Education}
+\subsection{\href{https://www.ucf.edu/}{UCF}}
+\descript{\href{https://www.ucf.edu/degree/information-technology-bs/}{B.S. in Information Technology}}
+\location{December 2014 | Orlando, Fl}
+\href{https://www.cecs.ucf.edu/}{College of Eng. \& Computer Sciences} \\
+Dean's List (Later Semesters) \\
+\location{Major GPA: 3.44 / 4.0}
 
 \sectionspace % Some whitespace after the section
 
-\descript{BS in Computer Science}
-\location{Expected May 2014 | Ithaca, NY}
-Conc. in Software Engineering \\
-College of Engineering \\
-Dean's List (All Semesters) \\
-\location{ Cum. GPA: 3.92 / 4.0 \\
-Major GPA: 3.94 / 4.0}
-
-\sectionspace % Some whitespace after the section
 
+%------------------------------------------------
+% Certifications
 %------------------------------------------------
 
-\subsection{La Martiniere for Boys}
-
-\location{Grad. May 2011 | Kolkata, India}
-
+\section{Certifications}
+\subsection{\href{https://www.giac.org/}{GIAC} (Expired)}
+2017 \href{https://www.giac.org/certifications/cyber-threat-intelligence-gcti/}{GCTI} (\#1144) \\
+2017 \href{https://www.giac.org/certifications/certified-forensic-analyst-gcfa/}{GCFA} (\#13368) \\
+2019 \href{https://www.giac.org/certifications/web-application-penetration-tester-gwapt/}{GWAPT} (\#6137) \\
 \sectionspace % Some whitespace after the section
 
 %------------------------------------------------
-% Links
+% Skills
 %------------------------------------------------
 
-\section{Links} 
+\section{Skills}
 
-Github:// \href{https://github.com/deedydas}{\bf deedydas} \\
-LinkedIn:// \href{https://www.linkedin.com/in/debarghyadas}{\bf debarghyadas} \\
-YouTube:// \href{https://www.youtube.com/user/DeedyDash007}{\bf DeedyDash007} \\
-Twitter:// \href{https://twitter.com/debarghya_das}{\bf @debarghya\_das} \\
-Quora:// \href{https://www.quora.com/Debarghya-Das}{\bf Debarghya-Das}
+\subsection{Programming}
+\href{https://aws.amazon.com/}{AWS} \textbullet{} Python \textbullet{} \textbf{\href{https://docs.panther.com/}{Panther}} \textbullet{} SQL \\
```

Comment:
(JC):
What’s bold mean (e.g., Panther, OCI)? I notice they are both links but then GCP isn’t linked.

Reply:
bold is meant to be links, fixed ty.